pkgs/tools/networking/networkmanager/applet/default.nix

```diff
@@ -19,7 +19,7 @@
 , gobject-introspection
 , python3
 , gtk3
-, libappindicator-gtk3
+, libayatana-appindicator-gtk3
```

Comment:
`libayatana-appindicator-gtk3` is now just an alias to `libayatana-appindicator`, so you should refer to that instead

Reply:
Thanks for the heads-up.